### PR TITLE
Master root r

### PR DESCRIPTION
**Pull request by omazapa**
Code from ROOT R project,
-> New features added. now you can use R functions in C++ very easy using the class TRFunctionImport, that have overloaded operators to use objects like functions that receives template arguments and return TRObjects.
-> Documentation updated in http://oproject.org/tiki-index.php?page=ROOT+R+Users+Guide#Import

Best Regards

### CODE REVIEW

**Comment by karies** [comment]
Hi Omar,

we have changed the documentation syntax to doxygen; you have reverted some of these changes in your patch. Could you fix that, please?

Cheers, Axel.

File: tutorials/r/Function.C [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+   
+   print(confint(fit));
+   
+   devnew("Fitting Regression");
+   plot(xdata,ydata);
+   
+   TRObject xgrid=seq(min(xdata),max(xdata),Label["len"]=10);
+   lines(xgrid,predict(fit,xgrid),Label["col"] = "green");
+   devoff();
+}
```
Comment (karies) [inline tutorials/r/Function.C]: Can you add a newline?

**Comment by omazapa** [comment]
Hi Axel,
Yes, I will to write the doc in doxygen.

Thanks you

**Comment by omazapa** [comment]
Hi Axel,

The fixes are ready and the documentation in doxygen is ready 
http://files.oproject.org/root/rootdoc/html/group___r.html
users guide in markdown integrate to doxygen 
http://files.oproject.org/root/rootdoc/html/md__home_omazapa_root_bindings_r_doc_users-guide__r_o_o_t_r__users__guide.html



Best regards

**Comment by omazapa** [comment]
Hi Axel and Lorenzo,

I have all updated, well tested  and ready for merge.
If you want to do other review and you have corrections please let me know.

On the other hand I have a question:
I have a users guide in markdown in $ROOTSYS/bindings/r/doc/users-guide and I can see that in the last commits you have the users guides in $ROOTSYS/documentation/users-guide
must I move my users guide there?

Best Regards!

**Pull request closed without merging**